stable/vpa/templates/recommender-deployment.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.recommender.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "vpa.fullname" . }}-recommender
+  labels:
+    app.kubernetes.io/component: recommender
+    {{- include "vpa.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.recommender.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: recommender
+      {{- include "vpa.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        app.kubernetes.io/component: recommender
+        {{- include "vpa.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "vpa.serviceAccountName" . }}-recommender
+      securityContext:
+        {{- toYaml .Values.recommender.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.recommender.securityContext | nindent 12 }}
+          image: "{{ .Values.recommender.image.repository }}:{{ .Values.recommender.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.recommender.image.pullPolicy }}
+          {{- if .Values.recommender.extraArgs }}
+          args:
+          {{- range $key, $value := .Values.recommender.extraArgs }}
+            - --{{ $key }}={{ $value }}
```

Comment:
maybe `quote` for safety

Reply:
```suggestion
            - --{{ $key }}={{ quote $value }}
```

Reply:
I tried that and it failed e2e tests, so I dropped it.